outboundSender.go

```diff
@@ -415,13 +416,15 @@ func (obs *CaduceusOutboundSender) Queue(msg *wrp.Message) {
 			}
 		*/
 		if matchDevice {
-			if len(obs.queue) < obs.queueSize {
+			select {
+			case obs.queue <- msg:
 				obs.queueDepthGauge.Add(1.0)
-				obs.queue <- msg
 				debugLog.Log(logging.MessageKey(), "WRP Sent to obs queue", "url", obs.id)
-				// a regex was matched, no need to check further matches
 				break
```

Comment:
Unfortunately, because the `break` is in a select statement, it now applies to the `select` rather than the for loop it originally applied to:
https://golang.org/ref/spec#Break_statements
I would suggest just removing the break (and the continue), and moving lines 428 and 429 inside of the `default` option for the select statement.  Since there is nothing that needs to apply to both cases, we don't need anything outside of the `select`.